shared/metrics/src/main/java/io/pravega/shared/metrics/Counter.java

```diff
@@ -25,11 +24,6 @@
      */
     void inc();
 
-    /**
-     * Decrement the value associated with this stat.
-     */
-    void dec();
```

Comment:
Any particular reason why we are removing decrements?

Reply:
I don't think we ever use them

Reply:
One principal of Counter is that it never goes down. Our existing implementation is anti-pattern, it decreases counter by resetting a new counter with a lower value.
Hence the dec() is removed from API to tight the discipline .